packages/newspringchurchapp/src/prayer/AddPrayer/AddPrayerCard/index.js

```diff
@@ -0,0 +1,49 @@
+import React, { memo } from 'react';
+import { Image } from 'react-native';
+import PropTypes from 'prop-types';
+import { H5, styled, Button, Card, CardContent } from '@apollosproject/ui-kit';
+import AddPrayerHeader from '../AddPrayerHeader';
+
+const StyledCardContent = styled(() => ({
+  alignItems: 'center',
+}))(CardContent);
+
+const Description = styled(({ theme }) => ({
+  color: theme.colors.text.secondary,
+  marginBottom: theme.sizing.baseUnit,
+}))(H5);
+
+const AddPrayerButton = styled(({ theme }) => ({
+  marginTop: theme.sizing.baseUnit * 2,
```

Comment:
Between the `marginTop` here and the `marginBottom` on the description I think we've got a little too much white space. I think just `baseUnit` here would be fine.
```suggestion
  marginTop: theme.sizing.baseUnit,
```